consensus/cometbft/service/options.go

```diff
@@ -81,3 +83,13 @@ func SetChainID[
 ](chainID string) func(*Service[LoggerT]) {
 	return func(s *Service[LoggerT]) { s.chainID = chainID }
 }
+
+// SetTargetBlockTime returns a Service option function that sets the desired
+// block time (e.g., 2s). Note that it CAN'T be lower than the minimal (floor)
+// block time in the network, which is comprised of the time to a) propose a
+// new block b) gather 2/3+ prevotes c) gather 2/3+ precommits.
+func SetTargetBlockTime[
+	LoggerT log.AdvancedLogger[LoggerT],
+](t time.Duration) func(*Service[LoggerT]) {
+	return func(bs *Service[LoggerT]) { bs.setTargetBlockTime(t) }
```

Comment:
note that we do enforce some floor on timeouts in https://github.com/berachain/beacon-kit/blob/9097bf316facfdee7d6adc9461776550fd7c44af/consensus/cometbft/service/configs.go#L127

Reply:
It is unclear to me how to do it from the API perspective. Do you have any suggestions @abi87?

Reply:
It's now a const, so enforcing the minimum is unnecessary.